tasks/packer-custom-image.yml

```diff
@@ -0,0 +1,20 @@
+---
+platform: linux
+
+image_resource:
+  type: registry-image
+  source: {repository: hashicorp/packer}
+
+params:
+  GCP_PROJECT:
+  GCP_KEY:
+
+inputs:
+- name: ci
+- name: linux-kernel-44
+
+outputs:
+- name: outputs
```

Comment:
Maybe use a less generic output name e.g. `packer-image`? And/or use `output_mapping` in the `task`?